.coderabbit.yaml

```
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: "en-US"
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)